jib-core/src/main/java/com/google/cloud/tools/jib/configuration/PortWithProtocol.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.configuration;
+
+/** Holds port number and protocol for an exposed port. */
+public class PortWithProtocol {
+
+  /** Represents the protocol portion of the port. */
+  public enum Protocol {
+    tcp,
```

Comment:
Oh enums should definitely be uppercase since they are constants. I meant that they should be instantiated with the string form, so like `TCP("tcp")`.